src/main/java/com/three_tech_solutions/slot_app/controllers/responses/StudentDetailsResponse.java

```diff
@@ -19,6 +19,7 @@ public record StudentDetailsResponse (
      LocalDate admissionDate,
      String paymentPlanName,
      String plan,
+     Integer numberOfDays,
```

Comment:
Acá podemos hacer que el  `numberOfDays` sea un byte en vez de un Integer, de esta manera evitamos tener que hacer el parseo a Integer en el `StudentMapper.java`.
¿Por qué usamos byte en vez de int?
Basicamente por el tamaño posible de los numeros, cuando nosotros en Java decimos que un atributo es de tipo int, en memoria estamos reservando un espacio de 4 bytes, mientras que si indicamos que es de tipo byte, reservamos 1 solo byte. Acá dejo un resumen de los numéricos de Java

<img width="1126" height="706" alt="Image" src="https://github.com/user-attachments/assets/5a1242a3-7208-4869-bb65-e720ab778042" />

Como podés ver, para indicar "el numero de días de clases que va el alumno por semana" nos alcanza y sobra por mucho con un solo byte (podemos marcar que va hasta 128 dias a la semana). En cambio con un int podríamos indicar que va a ir 2,1 mil millones de veces por semana, imposible

Son cosas bobas pero que aportan eficiencia al sistema, por eso usamos byte =)


```suggestion
     byte numberOfDays,
```